### PR TITLE
Pic of the day

### PR DESCRIPTION
**Pull request by BelaM80**

### CODE REVIEW

File: client/src/components/ModalDay/HeaderModalDay.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  setModalDayOpen,
+  modalDayOpen,
+  onSubmit,
+  value,
+  onChange
+) => {
+  return (
+    <Header>
+      <button onClick={() => setModalDayOpen(!modalDayOpen)}>✖️</button>
+      <form onSubmit={onSubmit}>
```
Comment (mhstrkmp) [inline client/src/components/ModalDay/HeaderModalDay.js:31]: Maybe missing a `event.preventDefault()`

File: client/src/components/ModalDay/InnerModalDay.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <p>{explanation}</p>
+      </div>
+    </Frame>
+  );
+};
+InnerModal.propTypes = {
+  title: PropTypes.string,
+  explanation: PropTypes.string,
+  imageSrc: PropTypes.node,
+  date: PropTypes.string,
```
Comment (mhstrkmp) [inline client/src/components/ModalDay/InnerModalDay.js:35]: Maybe missing `isRequired` at propTypes

File: client/src/components/ModalDay/ModalDay.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+};
+
+ModalDay.propTypes = {
+  modalDayOpen: PropTypes.bool,
+  setModalDayOpen: PropTypes.func,
+  nasaPic: PropTypes.object,
+  value: PropTypes.string,
+  onChange: PropTypes.func,
+  onSubmit: PropTypes.func,
+};
```
Comment (mhstrkmp) [inline client/src/components/ModalDay/ModalDay.js:71]: Maybe missing `isRequired` at propTypes

**Review by mhstrkmp — APPROVED** [review]
lgtm 🚀

File: client/src/components/ModalDay/HeaderModalDay.js (hunk 1 above)
Comment (BelaM80) [reply client/src/components/ModalDay/HeaderModalDay.js:35]: It´s there, but it´s defined in the page file, where the onSubmit ist written. 😊

File: client/src/components/ModalDay/ModalDay.js (hunk 3 above)
Comment (BelaM80) [reply client/src/components/ModalDay/ModalDay.js:70]: Thank you for your review! 😊 🎉

**Pull request merged**